coverage-tests/configuration/index.js

```diff
@@ -1,8 +1,11 @@
 module.exports = {
   name: 'eyes_selenium_java',
   emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/emitter.js',
-  overrides: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/overrides.js',
+  overrides: [
+    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/overrides.js',
+  ],
   template: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/template.hbs',
+  tests: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/coverage-tests.js',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.java3/pullRequest?prid=7879168)
```suggestion
  tests: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/coverage-tests.js",
```